rethinkdb_base_block.py

```diff
@@ -0,0 +1,94 @@
+import rethinkdb as rdb
+
+from nio.block.base import Block
+from nio.command import command
+from nio.properties import (IntProperty, StringProperty, VersionProperty,
+                            TimeDeltaProperty)
+from nio.util.discovery import not_discoverable
+
+
+@command("Get server info", method="_get_server_info")
+@command("Reconnect to server", method="_reconnect_connection")
+@command("List server databases", method="_get_list_of_databases")
+@command("List database tables", method="_get_list_of_tables")
```

Comment:
I don't have a good reason for it, but it doesn't feel like the blocks should be used in this way. To temporarily use a RethinkDB block just to run some commands to inspect a ReDB server doesn't seem like a good workflow.